tests/test_contact_specific_functions.py

```diff
@@ -13,54 +14,97 @@
 )
 
 
+def mock_rod_init(self):
+
+    "Initializing Rod"
+
+    """This is a small rod with 2 elements;
+    Initial Parameters:
+    radius = 1, length = 0.5,
+    tangent vector for both elements is (1, 0, 0),
+    stationary rod i.e velocity vector of each node is (0, 0, 0),
+    internal/external forces vectors are also (0, 0, 0)"""
+
+    self.n_elems = 2
+    self.position_collection = np.array([[1, 2], [0, 0], [0, 0]])
```

Comment:
Can you adjust dimensions such that quantities on nodes such as position, velocity and forces have dimension 3,n_elem+1 ?